src/servers/src/opentsdb/codec.rs

```diff
@@ -144,8 +144,9 @@ impl DataPoint {
         line_writer.finish()
     }
 
-    // TODO(fys): will remove in the future.
-    pub fn as_grpc_insert(&self) -> InsertExpr {
+    // TODO(LFC): opentsdb and influxdb insertions should go through the Table trait directly.
```

Comment:
We need to create or alter the table on demand. It might be difficult to bypass the gRPC layer.

Reply:
I agree. Will find the most suitable way to deal with it.